<launch>

<group ns="robot_p">
  <include file="$(find robot_p)/launch/urdf.launch" />

  <node pkg="robot_p" type="robot_control" name="robot_control">
    <rosparam>
       max_angular_velocity: 2
       serial_port: "/dev/ttyS0"
       Kp:1
       Ki:0.3
       Kd:0.5
    </rosparam>
  </node>

  <rosparam command="load" file="$(find robot_p)/config/control.yaml" />

  <node name="controller_spawner" pkg="controller_manager" type="spawner"
        args=" robot_p_diff_drive_controller --shutdown-timeout 1" />
</group>
</launch>
